admin-portal/src/app/_components/OrganizationSignUpForm.tsx

```diff
@@ -0,0 +1,94 @@
+"use client";
+
+import { useState } from 'react';
+import axios from "axios";
+
+
+export default function NewOrganizationForm() {
+    const [community, setCommunity] = useState('');
+    const [description, setDescription] = useState('');
+    const [location, setLocation] = useState('');
+    const [name, setName] = useState('');
+    const [password, setPassword] = useState('');
+    const [confirmPassword, setConfirmPassword] = useState('');
```

Comment:
For each of these state variables, define the type of each of them:
ex. useState<your_type_here>('');

This will help the compiler determine any errors whenever we build for debugging purposes and other static checks.